packages/sdk/src/__generated__/tags.ts

```diff
@@ -1,4 +1,6 @@
 export const tags: string[] = [
```

Comment:
this was sorted alphabetically before, the need to have div/span first is unobvious and you can't rely on alphabetic order any more in case something was expecting alphabetic

Reply:
Nothing relied on it. We just give div and span more priority when auto insert